language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/FunctionGenerator.java

```diff
@@ -164,10 +166,11 @@ public static String generateFunction(DocumentServiceContext context, boolean ne
      * @param returnsClause  Returns clause
      * @param returnStmt     Return statement
      * @param newLineAtStart Whether to add a new line at the start of the function
+     * @param isolated       Whether the created function should be prefixed with isolated qualifier
      * @return Created function
      */
     private static String generateFunction(String functionName, List<String> args, String returnsClause,
```

Comment:
If we have a function signature like this, when we expect a function signature without `isolated` qualifier, we have to include `false` explicitly. What if we provide a separate method to generate the isolated function?

Reply:
Done